tests/python/gaia-ui-tests/gaiatest/apps/bzlite/app.py

```diff
@@ -20,16 +30,59 @@ def login(self, username, password):
         password_element.tap()
         password_element.send_keys(password)
         button_element = self.marionette.find_element(*self._button_login)
-        button_element.tap()   
+        button_element.tap()
+
+    def logout(self):
+        profile_element = self.marionette.find_element(*self._profile_icon)
+        profile_element.tap()
+        logout_element = self.marionette.find_element(*self._button_logout)
+        logout_element.tap()
+
+    def followBugs(self):
+        flagged_element = self.marionette.find_element(*self._flagged_icon)
+        flagged_element.tap()
+        filed_element = self.marionette.find_element(*self._filed_icon)
+        filed_element.tap()
+        assigned_element = self.marionette.find_element(*self._assigned_icon)
+        assigned_element.tap()
+
+    def search(self, number):
+        searchbar_element = self.marionette.find_element(*self._search_bar)
+        searchbar_element.tap()
+        number_to_enter = self.keyboard.send(number)
+        searchbar_element.send_keys(number_to_enter)
+
+    def cancel_search(self):
+        searchbar_element = self.marionette.find_element(*self._search_bar)
+        searchbar_element.tap()
+        cancelsearch_element = self.marionette.find_element(*self._button_cancel_search)
+        cancelsearch_element.tap()
+
+    def create_new_bug(self, title, description):
```

Comment:
This method is a bit too big. This is a sign that there is something we can improve on the architecture side. I'd suggest to create a `PageRegion` for the 'Create bug' panel. [See this example](https://github.com/mozilla-b2g/gaia/blob/master/tests/python/gaia-ui-tests/gaiatest/apps/contacts/regions/contact_form.py).